packages/superset-ui-core/src/components/SafeMarkdown.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ReactMarkdown, { MarkdownAbstractSyntaxTree } from 'react-markdown';
+// @ts-ignore no types available
+import htmlParser from 'react-markdown/plugins/html-parser';
+
+import { FeatureFlag, isFeatureEnabled } from '../index';
```

Comment:
```suggestion
import { FeatureFlag, isFeatureEnabled } from '../utils';
```

In general I think imports within a library should be more explicit to save any future trouble of circular imports.